mne_bids/mne_bids.py

```diff
@@ -485,30 +484,20 @@ def _sidecar_json(raw, task, manufacturer, fname, kind, eeg_ref=None,
     return fname
 
 
-def raw_to_bids(subject_id, task, raw_file, output_path, session_id=None,
-                acquisition=None, run=None, kind='meg', events_data=None,
-                event_id=None, hpi=None, electrode=None, hsp=None,
-                eeg_ref=None, eeg_gnd=None, config=None, overwrite=True,
-                verbose=True):
+def write_raw_bids(raw, bids_fname, output_path, kind='meg', events_data=None,
+                   event_id=None, eeg_ref=None, eeg_gnd=None, overwrite=True,
+                   verbose=True):
```

Comment:
I'm imagining an analogous function called `read_raw_bids` for our `0.2` release